src/analyzer/protocol/pop3/POP3.cc

```diff
@@ -673,6 +673,7 @@ void POP3_Analyzer::ProcessReply(int length, const char* line) {
                         masterState = detail::POP3_UPDATE;
 
                     break;
+                default: break;
```

Comment:
Is there some standard way to mark this as "unreachable" so it's clear this is to please a tool?

Same below.

Reply:
There is `__builtin_unreachable()`. It's a compiler intrinsic that I know gcc and clang support. I'll need to see if it's supported by msvc though.